src/service.js

```diff
@@ -201,7 +208,14 @@ function processWantlist(context) {
       continue
     }
 
-    processEntry(batch[i], context)
+    if (trackResponseTime) {
```

Comment:
I have to check, but it looks an heavy operation to me, so usually should be disabled